### PR TITLE
doc: update the usage of 'ceph-deploy purge'

### PR DESCRIPTION
**Pull request by hikycool**
Now it refuses to purge data directly before remove ceph package.
So update the order between commands when using it.

Signed-off-by: Yu Shengzuo <yu.shengzuo@99cloud.net>

### CODE REVIEW

**Comment by tchaikov** [comment]
@hikycool could you attach the failure message when `ceph-deploy` refuses to purge data without uninstalling ceph in the comment or better off in the commit message?

**Comment by hikycool** [comment]
Here are the logs:
[root@syu-devstack ceph]# ceph-deploy purgedata syu-ceph2
[ceph_deploy.install][ERROR ] Ceph is still installed on: ['syu-ceph2']
[ceph_deploy][ERROR ] RuntimeError: refusing to purge data while Ceph is still installed

File: doc/start/quick-ceph-deploy.rst [hunk 1]
```diff
@@ -40,13 +40,10 @@ Create a Cluster
 If at any point you run into trouble and you want to start over, execute
 the following to purge the configuration::
 
+	ceph-deploy purge {ceph-node} [{ceph-node}]
```
Comment (tchaikov) [inline doc/start/quick-ceph-deploy.rst]: > to purge the configuration

this is not accurate. @johnwilkins shall we reword it to something like, 

> to purge the Ceph package, and erase all its data and configurations::

?

Comment (hikycool) [inline doc/start/quick-ceph-deploy.rst]: Hh～, that is better. I will fix it soon.

And could you know why 'cmake'  can't compile <pthread> or <venctor> and some other head file when I have installed all dependences. Something wrong in Makefile ?
OS : centos7
cmake:2.8.11
Please teach  me, I have been troubled by this for a long time。Thanks!

Comment (tchaikov) [inline doc/start/quick-ceph-deploy.rst]: @hikycool maybe you could shoot this question over IRC or mailing list with more details. as it's not relevant here. and i have no idea the error you have while configuring/building ceph.

Comment (hikycool) [inline doc/start/quick-ceph-deploy.rst]: OK,  thanks!

File: doc/start/quick-ceph-deploy.rst [hunk 2]
```diff
@@ -38,15 +38,12 @@ Create a Cluster
 ================
 
 If at any point you run into trouble and you want to start over, execute
-the following to purge the configuration::
+the following to purge the Ceph package, and erase all its data and configuration::
```
Comment (tchaikov) [inline doc/start/quick-ceph-deploy.rst]: s/package/packages/

Comment (hikycool) [inline doc/start/quick-ceph-deploy.rst]: Yes , it should be 'packages'.
def uninstall(distro, purge=False):
    packages = [
        'ceph',
        'ceph-release',
        'ceph-common',
        'ceph-radosgw',
        ]
Thank you!

**Comment by hikycool** [comment]
Thanks!


Original Message
Sender:Kefu Chainotifications@github.com
Recipient:ceph/cephceph@noreply.github.com
Cc:Yu Shengzuo466427645@qq.com; Mentionmention@noreply.github.com
Date:Wednesday, May 17, 2017 16:54
Subject:Re: [ceph/ceph] doc: update the usage of 'ceph-deploy purge' (#15080)


Merged #15080.
